### PR TITLE
Feature/#99add fake resume modality endpoint

### PR DESCRIPTION
**Pull request by pupadevs**
### Description
This PR adds the `CourseModality` controller to handle course modalities and corresponding Swagger documentation.

### Proposed Changes
- Created the `CourseModality` controller with an `invoke` method to display the student's modality.
- Obtained modality data from the `modality.json` file for initial testing (hardcoded).
- Added Swagger documentation for the endpoint related to course modalities.

### Tests Performed
- Conducted unit tests for the `invoke` method of the `CourseModality` controller.

### CODE REVIEW

**Review by coxen34 — APPROVED** [review]
As a fake endpoint to provide data, I see it as correct. 
But, is it the modality of the course? Or the modality of work, are you looking for?

File: app/Annotations/OpenApi/AnnotationsInfo.php
Comment (FranEnLaNube) [inline app/Annotations/OpenApi/AnnotationsInfo.php:1]: This is going to be removed once the new #108 is going to be merged.

File: app/Http/Controllers/api/CourseModalityController.php
Comment (FranEnLaNube) [inline app/Http/Controllers/api/CourseModalityController.php:1]: We should take care in the future to don't forget to delete this randomization for "modality"

**Review by FranEnLaNube — APPROVED** [review]
Good job!

**Pull request merged**